skimage/restoration/_denoise.py

```diff
@@ -110,10 +110,13 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
     sigma_color : float
         Standard deviation for grayvalue/color distance (radiometric
         similarity). A larger value results in averaging of pixels with larger
-        radiometric differences. Note, that the image will be converted using
-        the `img_as_float` function and thus the standard deviation is in
-        respect to the range ``[0, 1]``. If the value is ``None`` the standard
-        deviation of the ``image`` will be used.
+        radiometric differences. If ``None``, the standard deviation of
+        ``image`` will be used. Note that, if the image is of
+        any `int` dtype, ``image`` will be converted using the `img_as_float`
+        function and thus the standard deviation will be in
+        range ``[0, 1]``. For more information on scikit-image's data
+        type conversions and how images are rescaled in these conversions,
+        see: https://scikit-image.org/docs/stable/user_guide/data_types.html.
```

Comment:
I think that `Note that... .html.` would be better placed in the `Notes` section, like:
```
    Note that, if the image is of
    any `int` dtype, ``image`` will be converted using the `img_as_float`
    function and thus the standard deviation will be in
    range ``[0, 1]``.

    For more information on scikit-image's data
    type conversions and how images are rescaled in these conversions,
    see: https://scikit-image.org/docs/stable/user_guide/data_types.html.

Reply:
Respecting PEP-8, of course :sweat_smile:

Reply:
By just adding the new lines it adds it to the notes rather than the description?

Reply:
got it. just applied the suggestions you made. thanks!